streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -76,6 +76,25 @@ public void tearDown() throws Exception {
     }
 
     @Test
+    public void shouldNotThrowExceptionOnRestoreWithBulkLoad() throws Exception {
```

Comment:
Regarding test name: in this test we would actually not trigger bulk load, right?